templates/deployments.yaml

```diff
@@ -1773,6 +1774,10 @@ spec:
         - name: st2-rbac-mappings-vol
           mountPath: /opt/stackstorm/rbac/mappings/
         {{- end }}
+        {{- if .Values.st2.overrides }}
+        - name: st2-overrides-vol
+          mountPath: /opt/stackstorm/overrides/
+        {{- end }}
```

Comment:
Can you use your helper template instead here?

Reply:
Used the template here